xwiki-commons-core/xwiki-commons-environment/xwiki-commons-environment-servlet/src/main/java/org/xwiki/environment/internal/ServletEnvironment.java

```diff
@@ -27,10 +27,12 @@
 import java.net.URL;
 
 import javax.inject.Singleton;
-import javax.servlet.ServletContext;
 
 import org.apache.commons.lang3.exception.ExceptionUtils;
 import org.xwiki.component.annotation.Component;
+import org.xwiki.jakartabridge.servlet.JakartaServletBridge;
+
+import jakarta.servlet.ServletContext;
```

Comment:
I wonder if we should add a special rule for `jakarta` imports, e.g., to put them after `javax`?

Reply:
It would indeed probably make sense. It's easy to configure in Eclipse, not sure how it works for IDEA.

Reply:
Easy to configure in IDEA, too, I think we provide the configuration with the configuration we provide for IDEA.

Reply:
Sent https://forum.xwiki.org/t/add-a-special-ordering-location-for-jakarta-imports/16228.